book/src/validator-manager-move.md

```diff
@@ -0,0 +1,181 @@
+having a backup of all validator keystores (e.g. the mnemonic).**
+
+## Simple Example
+
+The following command will move all validators from the VC running at
+`http://localhost:6062` to the VC running at `http://localhost:5062`.
+
+```bash
+lighthouse \
+	validator-manager \
+	move \
+	--src-vc-url http://localhost:6062 \
+	--src-vc-token ~/src-token.txt \
+	--dest-vc-url http://localhost:5062 \
+	--dest-vc-token ~/.lighthouse/mainnet/validators/api-token.txt \
+	--validators all
+```
+
+## Detailed Guide
+
+This guide describes the steps to move validators between two validator clients (VCs) which are
+able to SSH between each other. This guide assumes experience with the Linux command line and SSH
+connections.
+
+There will be two VCs in this example:
+
+- The *source* VC which contains the validators/keystores to be moved.
+- The *destination* VC which is to take the validators/keystores from the source.
+
+There will be two hosts in this example:
+
+- Host 1 (*"source host"*): Is running the `src-vc`.
+- Host 2 (*"destination host"*): Is running the `dest-vc`.
+
+The example will assume that all commands are run on Host 1. It also assumes
+that Host 1 is able to SSH to Host 2.
+
+In reality, many host configurations are possible. For example:
+
+- Both VCs on the same host.
+- Both VCs on different hosts and the `validator-manager` being used on a third host.
+
+### 1. Configure the Source VC
+
+The source VC needs to have the following flags at a mininum:
+
+- `--http`
+- `--unencrypted-http-transport`
+- `--http-address 127.0.0.1`
+- `--http-port 5062`
+- `--http-allow-keystore-export`
+
+Therefore, the source VC command might look like:
+
+```bash
+lighthouse \
+    vc \
+    --http \
+    --unencrypted-http-transport \
+    --http-address 127.0.0.1 \
+    --http-port 5062 \
+    --http-allow-keystore-export
+```
+
+### 2. Configure the Destination VC
+
+The destination VC needs to have the following flags at a mininum:
+
+- `--http`
+- `--unencrypted-http-transport`
+- `--http-address 127.0.0.1`
+- `--http-port 5062`
+- `--enable-doppelganger-protection`
+
+Therefore, the destination VC command might look like:
+
+```bash
+lighthouse \
+    vc \
+    --http \
+    --unencrypted-http-transport \
+    --http-address 127.0.0.1 \
+    --http-port 5062 \
+    --enable-doppelganger-protection
+```
+
+> The `--enable-doppelganger-protection` flag is not *strictly* required, however
+> it is recommended for an additional layer of safety. It will result in 3-4
+> epochs of downtime for the validator after it is moved, which is generally an
+> inconsequential cost in lost rewards or penalties.
+> 
+> Optionally, users can add the `--http-store-passwords-in-secrets-dir` flag if they'd like to have
+> the import validator keystore passwords stored in separate files rather than in the
+> `valdiator-definitions.yml` file. If you don't know what this means, you can safely omit the flag.
+
```

Comment:
Added the configurations required if both VCs are in the same host.
````suggestion
> If both VCs are on the same host, it would required a different `http-port` value, e.g., `--http-port 5063`, and the command will become:
```bash
lighthouse \
    vc \
    --http \
    --unencrypted-http-transport \
    --http-address 127.0.0.1 \
    --http-port 5063 \
    --enable-doppelganger-protection
```
````